prep-javascript-variables/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = "Cody Campbell"
+    console.log('fullName:', fullName);
+    var numberOfPets = "1"
+    console.log('numberOfPets:', numberOfPets);
+    var likesVegetables = "false"
+    console.log('likesVegetables:', likesVegetables);
+    var nothing = "null"
```

Comment:
for numbers, booleans, and `null`, you should leave out the quotation marks. Any value in quotation marks is a string in javascript, but numbers, booleans, and `null` are their own values.